docs/repo_structure.md

```diff
@@ -52,3 +52,16 @@ These are plugins used in playbooks and roles:
 .
 └── test                Contains tests.
 ```
+
+### CI
+
+These files are used by [PAPR](https://github.com/projectatomic/papr),
+It is very similar in workflow to Travis, with the test
+environment and test scripts defined in a YAML file.
+
+```
+.
+├── .papr.yml
+├── .papr.sh
+└── .papr.inventory
```

Comment:
Heh, it's not empty, it was just renamed without any changes, so there's no diff ➕/➖ to show. The file is there: https://github.com/jlebon/openshift-ansible/blob/pr/papr/.papr.inventory.